src/saltext/vmware/modules/esxi.py

```diff
@@ -13,6 +13,8 @@
 from config_modules_vmware.esxi.esx_context import EsxContext
 from config_modules_vmware.lib.common.credentials import SddcCredentials
 from config_modules_vmware.lib.common.credentials import VcenterCredentials
+from config_modules_vmware.schema.schema_utility import Product
```

Comment:
The lines could be as follows:
```
from config_modules_vmware.schema.schema_utility import Product, retrieve_reference_schema
```
what oyu have is technically correct, but tendency is to put it all on the one line

Reply:
As discussed resolving this comment, since pre-commit is splitting these imports into separate lines.

```
Reorder python imports...................................................Failed
- hook id: reorder-python-imports
- exit code: 1
- files were modified by this hook

Reordering imports in tests/unit/modules/test_esxi.py
Reordering imports in src/saltext/vmware/modules/esxi.py
```